packages/@sanity/desk-tool/src/panes/documentPane/statusBar/documentSparkline.tsx

```diff
@@ -116,84 +172,89 @@ export function DocumentSparkline({badges, lastUpdated, editState}: DocumentSpar
     lastUnpublishOrPublishSession?.type === 'publish' && !isLiveDocument
 
   return (
-    <ContainerQuery className={styles.root} data-disabled={showingRevision}>
-      {showPublishedSessionBadge && (
-        <div className={styles.primarySessionBadgeContainer}>
-          <SessionBadge
-            type="publish"
-            icon={PublishIcon}
-            title={formatTimelineEventLabel('publish')}
-          />
-          <div className={styles.statusDetails}>
-            <div className={styles.label}>Published</div>
-            {lastPublishedTimeAgo && <div>{lastPublishedTimeAgo}</div>}
-          </div>
-        </div>
-      )}
-
-      {isLiveDocument && (
-        <div className={styles.primarySessionBadgeContainer}>
-          <SessionBadge type="live" title="Live document" icon={LiveIcon} />
-          <div className={styles.statusDetails}>
-            <div className={styles.label}>Published</div>
-            {lastUpdated && <div>{lastUpdatedTimeAgo}</div>}
-          </div>
-        </div>
-      )}
-
-      {!isLiveDocument && (
-        <Button
-          kind="simple"
-          padding="small"
-          onClick={openHistory}
-          type="button"
-          disabled={showingRevision || showingChangePanel}
-          className={styles.reviewChangesButton}
-          data-syncing={syncState.isSyncing}
-          title="Review changes"
-          data-transition={filteredSessions.length === 0 ? 'out' : transitionDirection}
-        >
-          <div className={styles.inner}>
-            <div
-              className={styles.sessionBadges}
-              style={{minWidth: sessionContainerWidth}}
+    <ResponsiveSparkline>
+      <ContainerQuery data-disabled={showingRevision}>
```

Comment:
Was unable to use the @sanity/ui ElementQuery because it didn't support the widths needed to make this work.